docker/django/Dockerfile

```diff
@@ -1,13 +1,13 @@
-#FROM python:3.4.3
-FROM container-registry.oracle.com/database/instantclient:12.2.0.1
+FROM python:3.4.3
```

Comment:
Another thing we need to start working on is to [rewrite the console](https://pythonclock.org/) to python 3

Reply:
The fact that this container seems to be working is encouraging, but yeah. Maybe something unexpected will fail when we try to add data here.